app/code/Magento/InventoryCatalog/Model/ResourceModel/GetSourcesCodesBySkus.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalog\Model\ResourceModel;
+
+use Magento\Framework\App\ResourceConnection;
+use Magento\Inventory\Model\ResourceModel\SourceItem;
+
+/**
+ * Retrieve a list of source codes by a selection of SKUs
+ */
+class GetSourcesCodesBySkus
```

Comment:
wrong naming 

GetSource`s`CodesBySkus


but in any case if we would decide to re-use existing service we can remove this class